### PR TITLE
Update service-tokens.md

### PR DESCRIPTION
**Pull request by amychen1776**
Update to make the service token creation flow align with the updated dbt Cloud inferface/naming.

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/service-tokens.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ You can assign as many permission sets as needed to one token. For more on permi
 
 ## Generating service account tokens
 
-In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new token. Create and save your token somewhere safe.
+To create a service token, go to the Account Settings page of dbt Cloud by clicking on the gear icon on the right hand side. Click on Service Tokens on the left sidebar and generate a new token. Create and save your token somewhere safe.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/service-tokens.md]: great suggestion @amychen1776 ! i've turned it into steps and bolded the UI elements! let me know what you think and I'll be happy to merge this for you!

```suggestion
To make a service token in dbt Cloud, follow these steps:

1. Open the **Account Settings** page by clicking the gear icon on the right-hand side.
2. On the left sidebar, click on **Service Tokens**.
3. Click the **+ New Token** button to generate a new token.
4. Once the token is generated, make sure to save it somewhere safe.
```

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
hey @amychen1776 ! wanted to check in on this and see if you were ok with the suggestion?

**Comment by amychen1776** [comment]
@mirnawong1 totally fine with it! Thank you and feel free to merge!

**Review by mirnawong1 — APPROVED** [review]
🚢

**Pull request merged**